content/enterprise_influxdb/v1.9/features/clustering-features.md

```diff
@@ -16,6 +16,22 @@ A valid license key is required in order to start `influxd-meta` or `influxd`.
 License keys restrict the number of data nodes that can be added to a cluster as well as the number of CPU cores a data node can use.
 Without a valid license, the process will abort startup.
 
+Access your license expiration date with the `/debug/vars` endpoint.
+In this examples we use `curl` and [`jq`](https://stedolan.github.io/jq/).
+
+{{< keep-url >}}
+```sh
+$ curl http://localhost:8086/debug/vars | jq '.entitlements'
+{
+  "name": "entitlements",
+  "tags": null,
+  "values": {
+    "licenseExpiry": "2022-02-15T00:00:00Z",
+    "licenseType": "license-key"
+  }
+}
+```
+
```

Comment:
I think the new `caption` shortcode would work well here for the handy, but not necessarily required information.

~~~suggestion
Access your license expiration date with the `/debug/vars` endpoint.

{{< keep-url >}}
```sh
$ curl http://localhost:8086/debug/vars | jq '.entitlements'
{
  "name": "entitlements",
  "tags": null,
  "values": {
    "licenseExpiry": "2022-02-15T00:00:00Z",
    "licenseType": "license-key"
  }
}
```
{{% caption %}}
This examples uses `curl` and [`jq`](https://stedolan.github.io/jq/).
{{% /caption %}}

~~~

Reply:
Thanks! 👍